reset-pass/account-validator-tool/src/webapp/css/passwordReset.css

```diff
@@ -169,27 +171,28 @@ table.myform {
 
 #strengthInfo
 {
-	display: block;
-	color: #000;
-	background-color: #fff;
-	border: 1px solid #ccc;
-	border-radius: 5px;
-	box-shadow: 1px 2px 3px rgba(0, 0, 0, 0.2);
-	left: 34em;
-	padding: 1em;
-	position: absolute;
-	margin-top: -5em;
-	width: 50%;
+    display: block;
```

Comment:
Just to further expand on Paul's point, the Sakai community is picky with regards to consistent whitespace in files; both on lines you have and have not changed functionally or syntactically.

So rule of thumb is to just use whatever is most prevalent in the file you're editing. If it's using spaces, use spaces; if it's using tabs, use tabs; if it's some crazy weird mixture of tabs and spaces, use that same crazy mixture....

The purpose of this nitpicking from the Community's standpoint is it helps to keep the Git history clean, as it becomes troublesome to figure out when and who made a change to specific lines if there are whitespace changes mixed in.